src/detectors/window_size.ts

```diff
@@ -1,9 +1,10 @@
 import { BotKind, ComponentDict, DetectorResponse, State } from '../types'
+import { getDocumentFocus } from '../utils/browser'
 
-export function detectWindowSize({ documentFocus, windowSize }: ComponentDict): DetectorResponse {
-  if (windowSize.state !== State.Success || documentFocus.state !== State.Success) return false
+export function detectWindowSize({ windowSize }: ComponentDict): DetectorResponse {
+  if (windowSize.state !== State.Success) return false
   const { outerWidth, outerHeight } = windowSize.value
   // When a page is opened in a new tab without focusing it right away, the window outer size is 0x0
-  if (!documentFocus.value) return
+  if (!getDocumentFocus()) return
```

Comment:
There is an edge case. User opens a page without focusing it, page collects sources. Detection `.detect` is executed after some time. User could focus the page within that time. So, we have `outer size 0` and `document focus`.

Reply:
it might be worth adding a note in readme with recommendations on how to call `detect()` for example inside `onfocus`:
```js
document.addEventListener("focus", function(){
  botd.detect()
})
```